danilo.tex

```diff
@@ -0,0 +1,185 @@
+
+We study the Helmholtz equation,
+
+\[
+\left\{
+\begin{array}{rc}
+    -\Delta u -k^2 u = f, &\Omega,\\
+    u=0, &\partial\Omega,
+\end{array}\right.
+\]
+
+for some $f\in H^{-1}(\Omega)$ and some boundary condition. Let's first explore what we can conclude with Lax-Milgram. 
+%Boundedness is trivial, so we look only at coercivity, which 
+We will assume in $H_0^1(\Omega)$ for simplicity.
+
+The weak form is:
+
+\[
+a(u,v) = (\nabla u, \nabla v) - k^2(u,v) = \langle f, v\rangle, \quad \forall v\in H_0^1.
+\]
+
+Using $H_0^1(\Omega)$ with the norm $\|v\|_{H^1_0} = \|\nabla v\|_0$, Poincaré inequality gives us $\|u\|_0\leq C_{\Omega}\|\nabla u\|_0$. 
```

Comment:
Acá por favor usa la notación que tenemos definida para la seminorma en H01 (|v|), cambiará un poco la redacción.

Reply:
Hola, no sé si me quedó claro como debo cambiar esta parte.

Reply:
Pon algo como: using H_0^1 with the norm induced by the H1 seminorm  $[...]$.